docs/application/dotnet/guides/nui/text.md

```diff
@@ -401,6 +401,23 @@ The following example illustrates text alignment:
 field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
+<a name="input-properties"></a>
+### Use Input Properties in TextField
+
+To change the text settings for new input text, you can use the Input properties of the [Tizen.NUI.BaseComponents.TextField](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextField.html) class, such as `InputColor`, `InputPointSize`, and so on.
+
+It is recommended to use these properties with some specific conditions. For instance, you can use these properties either when the `TextChanged` event occurs or when the `Clicked` event of button occurs.

```

Comment:
Thanks for the updates and review.

I have one question at here,  "It is recommended to use ..."

With https://github.com/Samsung/tizen-docs/pull/1287#discussion_r506001574,
I thought the properties "must" be used with some specific conditions, as the essential.

> This sentence is a key point. Input properties should be used with signals or any conditions.

However the sentence is changed to "It is recommended to use ..", as the nuance of optional.
Can we keep the current sentence?

Reply:
Hmm You're right.
It's not optional, but essential.
I'll change it again to 
-> To use these properties there are some specific conditions.